Soccar [UI]/src/nl/socnet/message/PlayerJoinedSessionMessage.java

```diff
@@ -1,39 +1,59 @@
 package nl.socnet.message;
 
 import nl.soccar.library.Player;

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'nl.soccar.library.Player'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUselessImportCheck)